ludwig/api.py

```diff
@@ -1425,7 +1425,9 @@ def preprocess(
 
             return PreprocessedDataset(proc_training_set, proc_validation_set, proc_test_set, training_set_metadata)
         except Exception as e:
-            raise RuntimeError(f"Caught exception during model preprocessing: {str(e)}") from e
+            raise RuntimeError(
+                f"Caught exception during model preprocessing: {str(e)}.\nFull traceback: {traceback.format_exc()}"
+            ) from e
```

Comment:
You should already get this with the `from e`, just need to scroll up a bit.

Reply:
Got it! Will revert this change.